### PR TITLE
Basic XML implementation

### PR DESCRIPTION
**Pull request by ghost**
This PR implements XML configuration file on a very basic level which allows to configure scraper in any way desirable. Headers management only at this time.

### CODE REVIEW

File: poland.py [hunk 1]
```diff
@@ -68,6 +69,52 @@ def get_core(content):
...
+  #
+  # Check whether file exists on the system or not and whether it's a file or a directory.
+  #
+  if not path.exists(xml_settings_file) or not path.isfile(xml_settings_file):
+    return "The file doesn't exist on the system."
+
+  #
+  # Retrieve data from XML file and place it inside of a variable
+  #
+  with open(xml_settings_file, 'r') as file:
```
Comment (Arr-n-D) [inline poland.py:84]: Should probably load in chunks as a large file will probably fuck this part up

**Review by Arr-n-D — CHANGES_REQUESTED** [review]

File: poland.py (hunk 1 above)
Comment (SapientLion) [reply poland.py:84]: Ok, i'll check that out.

**Comment by SapientLion** [comment]
@Arr-n-D I've made another version of my implementation. Please, take a look.

**Comment by xlanor** [comment]
Structure changed in master after refactoring, can this branch be updated with the new structure from master?

**Comment by SapientLion** [comment]
> Structure changed in master after refactoring, can this branch be updated with the new structure from master?

Done.

**Comment by SapientLion** [comment]
I'll put this PR on hold indefinitely until a better solution is found.

**Pull request closed without merging**